kubernetes-schema.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE "Cluster" (
+  "cluster_id" int PRIMARY KEY,
+  "name" varchar(255)
+);
+
+CREATE TABLE "Node" (
+  "node_id" int PRIMARY KEY,
```

Comment:
Could you put a link to the Resource Definition in the Kubernetes API as a comment, so one can quickly jump and see which other fields might belong here?